App/features/photos/sagas.ts

```diff
@@ -22,16 +23,49 @@ function * toggleStorage(action: ActionType<typeof PreferencesActions.toggleStor
   }
 }
 
-function * nodeOnline() {
-  while (yield take([getType(TextileEventsActions.nodeOnline), getType(TextileEventsActions.stopNodeAfterDelayCancelled)])) {
+function * bootstrapPhotoProcessing() {
+  const QUEUE_CONCURRENT = 1
+  const { watcher, addTaskChannel } = yield createQueue(photoHandler, QUEUE_CONCURRENT)
```

Comment:
This creates our processing queue and workers, provides a channel to add new photo processing tasks to